README.md

```diff
@@ -41,6 +42,15 @@ NOTE: Please make sure you configure the appropriate container runtime based on
 
 ## Usages
 
+### Use GenAI Microservices Connector(GMC) to deploy and adjust GenAIExamples
+
+Follow [GMC README](https://github.com/opea-project/GenAIInfra/blob/main/microservices-connector/README.md)
+to install GMC into your kubernetes cluster. GMC provides sample config for GenAIExamples under `microservices-connector/config/samples`. Select the example workflow you want to deploy, deploy the example (e.g. DocSum) using `kubectl`:
```

Comment:
Is my understanding correct that in samples we will provide mega language related samples for features such as switch, sequence, parallel etc? These could just point to existing samples that live in GenAIExamples but a smaller simpler sample may be better to explain how to use.

Should we take this opportunity to point folks to GenAIExamples OPEA sub-project versus a sub-directory here that we plan to remove?

s/GMC provides sample config for GenAIExamples under `microservices-connector/config/samples`. Select the example workflow you want to deploy, deploy the example (e.g. DocSum) using `kubectl`:
/GenAIExamples contains several sample GenAI example use case pipelines such as ChatQnA, DocSum, etc.
Once you have deployed GMC in your Kubernetes cluster, you can deploy any of the example pipelines by following its Readme file. 
.. give path to docsum kubernetes readme file ..